### PR TITLE
Compatible with new hardware layouts! -melgeek/z70ultra

### PR DESCRIPTION
**Pull request by mj-bomb**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/melgeek/z70ultra/z70ultra.h [hunk 1]
```diff
@@ -19,19 +19,19 @@
 #include "quantum.h"
 
 #define XXX KC_NO
-
+#ifdef REV1
```
Comment (drashna) [inline keyboards/melgeek/z70ultra/z70ultra.h:22]: these should go into the revision folders, not here.

File: keyboards/melgeek/z70ultra/rev2/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = STM32F303
+BOARD = QMK_PROTON_C
```
Comment (drashna) [inline keyboards/melgeek/z70ultra/rev2/rules.mk:3]: Is this actually using proton c? 

If not, it should be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/melgeek/z70ultra/z70ultra.h (hunk 1 above)
Comment (mj-bomb) [reply keyboards/melgeek/z70ultra/z70ultra.h:22]: I also tried some methods, but they failed to compile, can you give me some guidance?

Comment (drashna) [reply keyboards/melgeek/z70ultra/z70ultra.h:22]: `keebio/iris` should be a good example.

**Pull request closed without merging**